src/commands/step-functions/instrument.ts

```diff
@@ -0,0 +1,232 @@
+import {CloudWatchLogs, StepFunctions} from 'aws-sdk'
+import {Command} from 'clipanion'
+
+import {
+  createLogGroup,
+  enableStepFunctionLogs,
+  getStepFunction,
+  listStepFunctionTags,
+  putSubscriptionFilter,
+  tagStepFunction,
+} from './aws'
+import {displayChanges, applyChanges} from './changes'
+import {TAG_VERSION_NAME} from './constants'
+import {
+  buildArn,
+  buildLogGroupName,
+  buildSubscriptionFilterName,
+  getStepFunctionLogGroupArn,
+  isValidArn,
+  parseArn,
+} from './helpers'
+import {RequestsByStepFunction} from './interfaces'
+
+const cliVersion = require('../../../package.json').version
+
+export class InstrumentStepFunctionsCommand extends Command {
+  public static usage = Command.Usage({
+    description: 'Subscribe Step Function Log Groups to a Datadog Forwarder',
+    examples: [
+      [
+        'View and apply changes to subscribe a Step Function Log Group to a Datadog Forwarder',
+        'datadog-ci step-functions instrument --step-function arn:aws:states:us-east-1:000000000000:stateMachine:ExampleStepFunction --forwarder arn:aws:lambda:us-east-1:000000000000:function:ExampleDatadogForwarder --env dev --service example-service',
+      ],
+      [
+        'View changes to subscribe a Step Function Log Group to a Datadog Forwarder',
+        'datadog-ci step-functions instrument --step-function arn:aws:states:us-east-1:000000000000:stateMachine:ExampleStepFunction --forwarder arn:aws:lambda:us-east-1:000000000000:function:ExampleDatadogForwarder --env dev --service example-service --dry-run',
+      ],
+      [
+        'View and apply changes to subscribe multiple Step Function Log Groups to a Datadog Forwarder',
+        'datadog-ci step-functions instrument --step-function arn:aws:states:us-east-1:000000000000:stateMachine:ExampleStepFunction1 --step-function arn:aws:states:us-east-1:000000000000:stateMachine:ExampleStepFunction2 --forwarder arn:aws:lambda:us-east-1:000000000000:function:ExampleDatadogForwarder --env dev --service example-service',
+      ],
+    ],
+  })
+
+  private dryRun = false
+  private environment?: string
+  private forwarderArn!: string
+  private service?: string
+  private stepFunctionArns: string[] = []
+
+  public async execute() {
+    let validationError = false
+    if (typeof this.forwarderArn !== 'string') {
+      this.context.stdout.write('[Error] --forwarder is required\n')
+
+      return 1
```

Comment:
L53-L56 is probably redundant since `isValidArn()` expects `this.forwarderArn` to be `string!`.

Reply:
I think it will be helpful for a user to know if the `--forwarder` parameter was not set at all. I used a similar approach for both the forwarder and for step functions:

`[Error] --forwarder is required`
`[Error] invalid arn format for --forwarder`

`[Error] must specify at least one --step-function`
`[Error] invalid arn format for --step-function`